### ISSUE TITLE
Module '"../../../../node_modules/element-plus/types"' has no exported member 'ElMessage'. Did you mean 'Message'?

### ISSUE THREAD

**Issue by lqsong** [element-plus/element-plus#688]
```ts
import { ElMessage } from 'element-plus';
```
Tips:
> Module '"../../../../node_modules/element-plus/types"' has no exported member 'ElMessage'. Did you mean 'Message'?Vetur(2724)

Other:
```ts
import { ElNotification } from 'element-plus';
.
.
.
import { El*** } from 'element-plus';
```
Tips:
> Module '"../../../../node_modules/element-plus/types"' has no exported member 'El***'. Did you mean '***'?Vetur(2724)

**Comment by xiaogemm** [comment]
import { Message } from 'element-plus'

**Comment by lqsong** [comment]
> import { Message } from 'element-plus'

[https://element-plus.gitee.io/#/dan-du-yin-yong](https://element-plus.gitee.io/#/dan-du-yin-yong)

> 单独引用
```ts
import { ElMessage } from 'element-plus';
```
> 此时调用方法为 ElMessage(options)。我们也为每个 type 定义了各自的方法，如 ElMessage.success(options)。并且可以调用 ElMessage.closeAll() 手动关闭所有实例。

**Comment by JeremyWuuuuu** [comment]
This is a issue with typing, instead of the functionality itself, and that was a known issue, and we are planning on fix it in the next release. Thx.

**Comment by JeremyWuuuuu** [comment]
The error was thrown by `Vetur`, because it cannot find the definition according to the import name, because we have exported the member with a `El` prefix, but no with the `*.d.ts` file, which provides type definition to the component.

**Comment by JeremyWuuuuu** [comment]
We might have a release tonight, but that won't have any effect on the functionality.

**Comment by zazzaz** [comment]
fixed in 1.0.1-alpha.23

you can use 
```
import { ElMessage } from 'element-plus';
```
